### PR TITLE
I added 3 targets to make each frameworks.

### PR DESCRIPTION
**Pull request by dairyo**
I added 3 targets to make each framework. The target names are followings:

* ThingIFCoreSDK
* ThingIFNonTraitSDK
* ThingIFTraitSDK

I want to use the target names as framework names like ThingIFCoreSDK.framework.

What do you think about target names and framework names?

### CODE REVIEW

File: ThingIFSDK/ThingIFCoreSDKTests/ThingIFCoreSDKTests.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
+//
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFCoreSDKTests/ThingIFCoreSDKTests.swift:1]: This file is not necessary. Please remove it.

File: ThingIFSDK/ThingIFNonTraitSDKTests/ThingIFNonTraitSDKTests.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
+//
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFNonTraitSDKTests/ThingIFNonTraitSDKTests.swift:1]: This file is not necessary too.

File: ThingIFSDK/ThingIFTraitSDKTests/ThingIFTraitSDKTests.swift [hunk 3]
```diff
@@ -0,0 +1,36 @@
+//
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFTraitSDKTests/ThingIFTraitSDKTests.swift:1]: please remove it too.

**Comment by dairyo** [comment]
We distribute SDK as 1 framework. I withdraw this PR.

**Pull request closed without merging**